src/python/pants/build_graph/target.py

```diff
@@ -315,12 +272,19 @@ def maybe_readable_combine_ids(cls, ids):
 
   @classmethod
   def _closure_predicate(cls, include_scopes=None, exclude_scopes=None, respect_intransitive=False):
```

Comment:
This method can be deleted, afaict. It's not called anywhere, and isn't a public API.

Reply:
That was my thought too.